src/core/Recoil_Node.js

```diff
@@ -92,14 +92,14 @@ function registerNode<T>(node: Node<T>): RecoilValue<T> {
     // TODO Need to figure out if there is a standard/open-source equivalent to see if hot module replacement is happening:
     // prettier-ignore
     // @fb-only: if (__DEV__) {
-      // @fb-only: const isAcceptingUpdate = require('__debug').isAcceptingUpdate;
-      // prettier-ignore
-      // @fb-only: if (typeof isAcceptingUpdate !== 'function' || !isAcceptingUpdate()) {
-        // @fb-only: expectationViolation(message, 'recoil');
-      // @fb-only: }
+    // @fb-only: const isAcceptingUpdate = require('__debug').isAcceptingUpdate;
```

Comment:
It seems that Prettier doesn't ignore whitespace before comments, so it is removing the whitespace here when it's called by `lint-staged` during `pre-commit`. Is this fine or will it cause a problem when using this code internally?

Reply:
We'll see if this gets converted to uncommented lines with `// @fb-only` at the end when importing to our repository or we need to do that manually.  But, either way, since that source of truth is uncommented, then prettier should fix it when we do the internal diff.  So, this shouldn't be an issue.  The OSS only lines can be for the opposite reason, but we think we have a workaround for those using `{}` blocks.